kerl

```diff
@@ -985,6 +990,14 @@ fail_do_build() {
     fi
 }
 
+uname_r_label() {
+    echo "uname -r"
```

Comment:
Are we sure this is the right command on all of our platforms?? probably is - but for md5 and sed we had to detect platform before we used a command like this...

Reply:
I can check further. At the same time, I'm not sure:

1. we're always using `\` (sometimes we are) where commands are executed
  1.1. I can later pull request for this, maybe (?)
2. we're using the `sed` options consistently (don't seem to be)
  2.1. I can later pull request for this, maybe (?)

Reply:
`uname` should be POSIX, and `-r` is defined as "Write the current release level of the operating system implementation."

Reply:
From what I could gather, `uname -r` is valid for Darwin, OpenBSD, FreeBSD, and a host of Linux systems.

Reply:
> 1. we're always using `\` (sometimes we are) where commands are executed

I don't remember the exact commit where it was introduced (think it was some kind of shellcheck clean up???) but it is supposed to make it so the command is executed in its own environment without environment variables or something?

>    1.1. I can later pull request for this, maybe (?)

Yeah maybe? low priority tho

Reply:
> I don't remember the exact commit where it was introduced

ShellCheck does not complain about this sort of stuff, AFAIK. If I remember correctly is was an actual bug somebody wanted to fix...

> Yeah maybe? low priority tho

I have no real priorities (I'm kinda going, with `kerl`, with oldest issues first - if that's any type of priority order), but these should mostly be mechanical changes 😄

Reply:
Ref: https://github.com/kerl/kerl/pull/363